README.md

```diff
@@ -75,77 +75,118 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Added Awake as option in translation bug template. Thanks @Aaron-Junker!
+- Adjusted description of inactive setting to improve distinguishing between the utility being disabled vs inactive. Thanks @niels9001!
 
 ### Color Picker
 
-- Fixed escape behavior so that only the fly-out is closed if active.
-- Fixed several accessibility issues related to screen reader functionality and general usage of Color Picker.
+- Fixed bug where changing RGB values doesn't update color. Thanks @niels9001!
+- Fixed accessibility issue with screen reader not announcing when "Copied to Clipboard" is activated.
+- Fixed accessibility issue where user could not hover the content of the info icon using a mouse. Thanks @niels9001!
+- Fixed color picker format order not being accessible via keyboard. Thanks @niels9001!
+- Fixed accessibility issue where screen reader announces incorrect name for "Editor color format" button and not announcing "Toggle switch" button at all. Thanks @niels9001!
 
 ### FancyZones
 
-- Fixed bug causing multi-monitor spanning errors.
-- Added minimum zone size limit to the settings.
-- Fixed issue where re-opened windows don't appear in previously assigned zone.
-- Fixed excluded apps setting to save on text change instead of when leaving focus.
-- Fixed corrupt/outdated plugins load crash. 
-- Fixed issue with FancyZones not working after computer goes to sleep.
-- Added screen reader confirmation to canvas editor when new zones are added.
+- Adjusted "Save and apply" editor button to adjust with text size for localizations. Thanks @niels9001!
+- Fixed "Create new layout" button visibility when in high contrast mode. Thanks @niels9001!
+- Fixed scaling quirks related to editor UI. Thanks @niels9001!
+- Fixed editor crashing when double clicking the "edit layout" button.
+- Fixed issue with editor crashing immediately after displaying zones.
+- Fixed bug when navigating editor options via keyboard where pressing enter on unselected Canvas option launches Grid editor instead.
+- Fixed issue where FancyZones would not restore Console Applications.
+- Fixed Canvas editor and Grid editor window heights. Thanks @niels9001!
+- Fixed crash due to KERNELBASE.dll.
+- Fixed FancyZone icons to be smoother at higher DPI settings. Thanks @niels9001!
+- Fixed crash when changing between zone layouts.
+- Fixed regression where FancyZones does not resize windows on layout change.
+- Adjusted layout settings to reset shortcut key after canceling changes on a particular layout.
+
+### File Explorer add-ons
+
+- Fixed issue where markdown files were still previewed even when "Enable Markdown" was unselected.
+
+### Image Resizer
+
+- Added warning that GIF files with animations may no correctly resize if the encoding used for the files is incompatible.
 
 ### Keyboard Manager
 
-- Fixed screen reader usage bugs to increase intuitiveness.
+- Updated old settings UI style on Keyboard Manager preview in OOBE. Thanks @niels9001!
```

Comment:
If i am correct, is this based on an issue that was made because kbm and awake images were overlooked in a earlier pr? If so, then I think these don't need to be mentioned (individually)

Reply:
Good catch. You're right.